Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/jvm/main.kt

```diff
@@ -219,6 +220,12 @@ private fun processCLib(args: Array<String>, additionalArgs: Map<String, Any> =
 
     val (nativeIndex, compilation) = buildNativeIndex(library, verbose)
 
+    // Our current approach to arm64_32 support is to compile armv7k version of bitcode
+    // for arm64_32. That's the reason for this substitution.
```

Comment:
Add TODO to fix it with next LLVM update?

Reply:
Ok. I'll also create task for it.